HousingRepairsOnlineApi/Startup.cs

```diff
@@ -52,13 +55,63 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddHousingRepairsOnlineAuthentication(HousingRepairsOnlineApiIssuerId);
+            services.AddTransient<ISaveRepairRequestUseCase, SaveRepairRequestUseCase>();
+
+            var cosmosContainer = GetCosmosContainer();
+
+            services.AddTransient<IRepairStorageGateway, CosmosGateway>(s =>
+            {
+                return new CosmosGateway(
+                    cosmosContainer, new IdGenerator()

```

Comment:
Registering `IIdGenerator` mapping to `IdGenerator` within services would remove the need for this lambda as all constructor parameters would be registered.

i.e. `services.AddTransient<IIdGenerator, IdGenerator>();`